test/e2e/admin_portal.go

```diff
@@ -73,7 +73,7 @@ var _ = Describe("Admin Portal E2E Testing", func() {
 	})
 
 	It("Should be able to populate cluster info panel correctly", func() {
-		const CLUSTER_INFO_HEADINGS = 10
+		const cluster_info_headings = 10
```

Comment:
Out of curiosity, why did you change this constant's naming convention?

Snake case is okay for all caps, but not for lower case const/variables. It should rather be `clusterInfoHeadings`.

Reply:
I will revert it. 
It was suggested by @AldoFusterTurpin https://github.com/Azure/ARO-RP/pull/2500#discussion_r1026381936

Reply:
Done!

Reply:
My fault, when I said "lowercase" I meant either camel case or pascal case (to remove the underscores).